libs/application/templates/parental-leave/README.md

```diff
@@ -1,5 +1,138 @@
-# Application Templates Parental Leave
+# Parental leave application
 
-This application template is handling request for parental leave. It handles connections with external API, like Vinnumálastofnun or Þjóðskrá to get the information needed to create an application.
+## Description
 
-This application contains all the steps for the parents, employers, institutions to process the request along the way, until it is completed.
+This application template allows applicants to apply for parental leave. For further reading about application templates, have a look at `libs/application/templates/reference-template/README.md`.
```

Comment:
Should be a relative link?

https://docs.devland.is/repository/gitbook-contribute#links